<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerClass</class>
 <widget class="QMainWindow" name="VideoPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VideoPlayerClass{
background-color: rgb(53, 53, 53);
}
QPushButton:!hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(120, 120, 120, 255),stop:0.495      rgba(39, 39, 39, 255),stop:0.505 rgba(39, 39, 39, 255),stop:1 rgba(120, 120, 120, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
QPushButton:hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(39, 39, 39, 255),stop:0.495      rgba(120, 120, 120, 255),stop:0.505 rgba(120, 120, 120, 255),stop:1 rgba(39, 39, 39, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#centralWidget{
background-color: rgb(53, 53, 53);
}
#closeButton
{
font: 75 11pt &quot;等线&quot;;
color: rgb(255, 255, 255);
background-color: rgb(53, 53, 53);
}

#playButton{background-color: rgba(255, 255, 255,0);}

#playButton:hover{image: url(:/VideoPlayer/Resources/PlayN.png);}

#playButton:!hover{image: url(:/VideoPlayer/Resources/PlayH.png);}

#playButton:pressed{image: url(:/VideoPlayer/Resources/PlayP.png);}

#pauseButton{background-color: rgba(255, 255, 255,0);}

#pauseButton:hover{image: url(:/VideoPlayer/Resources/PauseN.png);}

#pauseButton:!hover{image: url(:/VideoPlayer/Resources/PauseH.png);}

#pauseButton:pressed{image: url(:/VideoPlayer/Resources/PauseP.png);}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoOpenGLWidget" name="vowScreen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="playSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>601</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videoopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VideoPlayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>openVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>setPos(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoPlayerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openVideo()</slot>
  <slot>pause()</slot>
  <slot>play()</slot>
  <slot>sliderRelease()</slot>
  <slot>sliderPress()</slot>
  <slot>setPos(int)</slot>
 </slots>
</ui>
